server/routes_test.go

```diff
@@ -104,6 +105,24 @@ func Test_Routes(t *testing.T) {
 				assert.Empty(t, len(modelList.Models))
 			},
 		},
+		{
+			Name:   "List Model Handler OpenAI (no tags)",
```

Comment:
For test names a better format would be lowercase:

```
openai list models no tags
```

Reply:
Do we want to make all the test names lowercase then?

Reply:
yes although no need to change them all here